src/config.ts

```diff
@@ -12,21 +12,21 @@ export const TAURI_BLUE = 0x67d6ed;
 const ADMIN_ROLES = DEV_MODE
 	? [process.env.DEV_ADMIN_ROLE]
 	: [
-			// admin
-			'985400380663935088',
-			// core
-			'616187491715907585',
-			// working-group
-			'761977421305610241',
-	  ];
+		// admin
```

Comment:
I swear the indentions in this file change every single PR 😂

Reply:
Yea I'm fairly sure it's related to me jumping between machines and VSCode is slightly differently set up so it auto formats the files a bit differently, and I'm just a tiiiiny bit too lazy to not just commit the updated formatting :sweat_smile: